track.py

```diff
@@ -218,6 +221,20 @@ def speedLimitsOk(self):
         return True if self.speedLimits.shape[0] > 0 and checkDataFrame(self.speedLimits, self.length) else False
 
 
+    def curvaturesOk(self):
+
+        if (self.curvatures['Curvature radius [m]'] < 0.0).values.any():
```

Comment:
I could not find where you take the absolute values of the curvatures. I would assume this should take place in the importCurvatures function, since json data are allowed to have negative values too